tutorials/Tutorial5_Evaluation.ipynb

```diff
@@ -931,327 +573,20 @@
-      " \toffsets_in_document: [{'start': 761, 'end': 770}]\n",
-      " \tcontext: niversity Orchestra at TEOC Classical Music Radio: Symphony No. 40 The apartment where Mozart wrote his last three Symphonies: Michael Lorenz, ``Mozar\n",
-      " \texact_match: 0.0\n",
-      " \tf1: 0.0\n",
-      " \trank: 3\n",
-      "\n",
-      " \tanswer: Greek\n",
-      " \tdocument_id: 5d1b1f7bd8bb212fa9e7b7f1508e5926-0\n",
-      " \toffsets_in_document: [{'start': 193, 'end': 198}]\n",
-      " \tcontext: an patronymic surname meaning ``son of Anders'' (itself derived from the Greek name ``Ανδρέας/Andreas'', cf. English Andrew). It is the fifth most com\n",
-      " \texact_match: 0.0\n",
-      " \tf1: 0.0\n",
-      " \trank: 4\n",
-      "_______________________________________________________\n",
-      "Query: \n",
-      " \twho took over for batman when his back was broken\n",
-      "Gold Answers: \n",
-      " \tJean-Paul Valley (a.k.a. Azrael)\n",
-      " \tan apprentice named Jean-Paul Valley (a.k.a. Azrael)\n",
-      "Gold Document Ids: \n",
-      " \t4c02586bad8a9fa0ef164d0b9bc27499-1\n",
-      " \t4c02586bad8a9fa0ef164d0b9bc27499-1\n",
-      "Metrics: \n",
-      " \texact_match: 0.0\n",
-      " \tf1: 0.0\n",
-      "Answers: \n",
-      " \tanswer: \n",
-      " \tdocument_id: None\n",
-      " \toffsets_in_document: [{'start': 0, 'end': 0}]\n",
-      " \tcontext: None\n",
-      " \texact_match: 0.0\n",
-      " \tf1: 0.0\n",
-      " \trank: 1\n",
-      "\n",
-      " \tanswer: Zoom\n",
-      " \tdocument_id: 4c02586bad8a9fa0ef164d0b9bc27499-29\n",
-      " \toffsets_in_document: [{'start': 6, 'end': 10}]\n",
-      " \tcontext: enemy Zoom overpowers Barry in their battle and breaks his back. He then goes to Central City News and the police department holding Barry's broken bo\n",
-      " \texact_match: 0.0\n",
-      " \tf1: 0.0\n",
-      " \trank: 2\n",
-      "\n",
-      " \tanswer: Bane\n",
-      " \tdocument_id: 4c02586bad8a9fa0ef164d0b9bc27499-30\n",
-      " \toffsets_in_document: [{'start': 636, 'end': 640}]\n",
-      " \tcontext: al wire and knocks out Bane with it. Film(edit) In Justice League: Doom, Bane makes reference to the storyline when he confronts Bruce Wayne above his\n",
-      " \texact_match: 0.0\n",
-      " \tf1: 0.0\n",
-      " \trank: 3\n",
-      "\n",
-      " \tanswer: Bane\n",
-      " \tdocument_id: 4c02586bad8a9fa0ef164d0b9bc27499-31\n",
-      " \toffsets_in_document: [{'start': 670, 'end': 674}]\n",
-      " \tcontext: tcave to challenge Batman. During this fight, if the player is caught by Bane he executes the infamous back breaker. The famous back-breaking scene wa\n",
-      " \texact_match: 0.0\n",
-      " \tf1: 0.0\n",
-      " \trank: 4\n",
-      "_______________________________________________________\n",
-      "=======================================================\n",
-      "\n"
-     ]
-    }
-   ],
+   "outputs": [],
```

Comment:
Here it was actually on purpose to have the output in the tutorial so that users can see the format of the Evaluation Report. I think we should keep it for that reason.

Reply:
Sure, I have included the output from just those cells which print the Evaluation Report